recipes/ruby/build.sh

```diff
@@ -0,0 +1,9 @@
+set -e
+set -x
+
+./configure --prefix=$PREFIX --disable-install-doc --enable-load-relative \
+  --enable-shared --with-openssl-dir=$PREFIX --with-readline-dir=$PREFIX \
+  --with-tcl-dir=$PREFIX --with-tk-dir=$PREFIX --with-libyaml-dir=$PREFIX \
+  --with-zlib-dir=$PREFIX
```

Comment:
It doesn't hurt  to have, though. That can be cleaned up in the feedstock if desired.